### PR TITLE
Fixes 2070 - Prevent crashes when inserting mention pills in the comp…

### PR DESCRIPTION
**Pull request by stefanceriu**
…osers on Mac

### CODE REVIEW

**Review by Velin92 — APPROVED** [review]

File: ElementX/Sources/Other/Pills/PillAttachmentViewProvider.swift [hunk 1]
```diff
@@ -30,8 +30,6 @@ protocol PillAttachmentViewProviderDelegate: AnyObject {
 final class PillAttachmentViewProvider: NSTextAttachmentViewProvider {
     private weak var delegate: PillAttachmentViewProviderDelegate?
     
-    // MARK: - Override
```
Comment (Velin92) [inline ElementX/Sources/Other/Pills/PillAttachmentViewProvider.swift:33]: I guess we can keep this line?

Comment (stefanceriu) [reply ElementX/Sources/Other/Pills/PillAttachmentViewProvider.swift:33]: If you really have strong feelings about it 😅 I figured all methods in there start with `override`, I might as well remove it

Comment (Velin92) [reply ElementX/Sources/Other/Pills/PillAttachmentViewProvider.swift:33]: well we know have the NSSecureCoding mark, so maybe we can have the override mark too to separate both of them? not really important, just point this out

Comment (stefanceriu) [reply ElementX/Sources/Other/Pills/PillAttachmentViewProvider.swift:33]: No no, that's fair, I'll add it back 👍

**Pull request merged**